packages/lib/src/core/presets/thumbnailGallery.js

```diff
@@ -35,6 +35,7 @@ export const createStyles = styles => {
   return {
     ...styles,
     ...fixedStyles,
+    targetItemSize: styles.targetItemSize || styles.gallerySize,
```

Comment:
Are these styles.gallerySize here so it wont break?
Otherwise if we decide its a breaking change of a rename we could have 0 gallerySize left in the code

Reply:
No - this is the place where the translation from `gallerySize` to `targetItemSize` is happening.
In this specific case, the translation is 1:1.
The translation from `gallerySizePx` and `gallerySizeRatio` to `targetItemSize` is happening in `processLayouts`